### PR TITLE
Use `FILESETS`/`REVSETS` for placeholders in command synopsis

### PR DESCRIPTION
**Pull request by martinvonz**
Thanks to @fowles and @yuja for the inspiration.

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Comment by cenviity** [comment]
Should we also change the accompanying doc comments then, e.g. path(s) / file(s) → fileset(s), revision(s) → revset(s)?

**Review by fowles — APPROVED** [review]

File: cli/src/commands/file/show.rs [hunk 1]
```diff
@@ -57,6 +58,7 @@ pub(crate) struct FileShowArgs {
         value_hint = clap::ValueHint::FilePath,
         add = ArgValueCompleter::new(complete::all_revision_files),
     )]
+    #[arg(value_name = "FILESETS")]
```
Comment (yuja) [inline cli/src/commands/file/show.rs:61]: nit: combine `arg()`s for consistency?

(applies to file/show.rs, file/untrack.rs, interdiff.rs, log.rs, restore.rs, squash.rs)

File: cli/src/commands/absorb.rs [hunk 2]
```diff
@@ -39,6 +39,7 @@ pub(crate) struct AbsorbArgs {
     #[arg(
         long, short,
         default_value = "@",
+        value_name="REVSET",
```
Comment (yuja) [inline cli/src/commands/absorb.rs:42]: nit: insert space around `=`? (there are a few more instances)

**Review by yuja — APPROVED** [review]

File: cli/src/commands/absorb.rs (hunk 2 above)
Comment (martinvonz) [reply cli/src/commands/absorb.rs]: Done.

File: cli/src/commands/file/show.rs (hunk 1 above)
Comment (martinvonz) [reply cli/src/commands/file/show.rs]: Oops, done. Thanks.

**Comment by martinvonz** [comment]
> Should we also change the accompanying doc comments then, e.g. path(s) / file(s) → fileset(s), revision(s) → revset(s)?

Yes, we should probably basically replace all "paths" by "paths matching these filesets" and "revisions" by "revisions matching these revsets". I'll try to remember to do that in a followup. I think this PR is an improvement on its own, and I should get back to the copy-tracking design.

**Pull request merged**